persdict.dat

```diff
@@ -62,3 +62,16 @@ sponte
 squacquerone
 subìto
 username
+snippet
+canvas
+WordPress
+popup
+LibreOffice
+evidenziazione
+color
+tablet
+Layout
+mozillians
+Contributor
```

Comment:
Usi mai "Contributor" con la prima lettera maiuscola come fosse inizio frase? È sicuramente sensato metterlo come prima lettera minuscola, ma in che situazione lo usi con maiuscola?